x-pack/plugins/security_solution/server/fleet_integration/handlers/create_default_policy.ts

```diff
@@ -83,9 +94,9 @@ const getEndpointPolicyWithIntegrationConfig = (
         },
       },
     };
+  } else {
+    return policy;
   }
-
-  return policy;
```

Comment:
I'm curious about the reason for changing the return to `else`?

Reply:
We have 3 ways to go here: only event, NGAV/EDREssential or EDRComplete. For me, these are three equivalent choices perfectly described by an `if`-`else if`-`else` statement, instead of handling some of them as special versus one default choice.

I think it aligns with the style guide's paragraph about [early return](https://docs.elastic.dev/kibana-dev-docs/contributing/styleguide#returnthrow-early-from-functions): there is no special case, no error to throw early, and the statement is not deeply nested.